model/api/src/main/java/org/opennms/oce/model/api/Model.java

```diff
@@ -47,4 +48,16 @@
      */
     ModelObject getRoot();
 
+    /**
+     * To support both batch and single operations of updating model. Using list in case if we need ordering
+     */
+    void updateObjects(List<ModelObject> moList);
```

Comment:
If another class is only depending on the `model/api` module, how do we expect these `ModelObject`s to be built?